### PR TITLE
Add UI back into same repo as service code

### PR DESCRIPTION
**Pull request by chrisrohr**
This will allow for an easier build and deployment structure going forward. In future updates, when we are ready to create distributions we can build a Docker image and not have to try to bring in multiple repos.

I have kept the sonar projects separate so the build will run sonar for service only and then for ui only since they don't get built all through Maven.

Service (maven/java) code lives in the directory service and UI (node/vue/quasar) code lives in the directory ui.

Updated the github actions to account for new directories and build processes for both "modules".

### CODE REVIEW

**Comment by chrisrohr** [comment]
This will close #65

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -46,21 +46,85 @@ jobs:
...
-        if: ${{ env.SONAR_TOKEN != null && env.SONAR_TOKEN != '' }}
+      - name: Analyze with SonarCloud on service
+        if: ${{ env.SERVICE_SONAR_TOKEN != null && env.SERVICE_SONAR_TOKEN != '' }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SERVICE_SONAR_TOKEN }}
         run: mvn -B -V -Dsonar.coverage.jacoco.xmlReportPaths=target/site/jacoco/jacoco.xml org.jacoco:jacoco-maven-plugin:prepare-agent verify org.jacoco:jacoco-maven-plugin:report org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+        working-directory: ./service
+  build-ui:
```
Comment (sleberknight) [inline .github/workflows/build.yml:70]: Could you just have separate build files and maybe there is a way to specify the `working-directory` for the entire file so it does not need to be in every step?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Chris Rohr
+Copyright (c) 2023 Kiwi Project
```
Comment (sleberknight) [inline LICENSE:3]: Huh. I guess I never added that license copyright year update workflow to this repo...remind me tomorrow.

File: ui/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (sleberknight) [inline ui/.editorconfig:1]: We need editorconfig files for all our other repos too!

File: ui/test/jest/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+coverage/*
```
Comment (sleberknight) [inline ui/test/jest/.gitignore:1]: make it happy...newline

File: ui/test/jest/__tests__/time.spec.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+});
+
+describe('formatDate', () => {
+    it ('returns the utc format for the given milliseconds', () => {
+        const now = moment();
+        const expected = now.utc().format('YYYY-MM-DD HH:mm:ss[Z]');
+
+        expect(formatDate(now.valueOf())).toBe(expected);
+    });
+});
```
Comment (sleberknight) [inline ui/test/jest/__tests__/time.spec.ts:22]: make it happy...newline

File: ui/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@quasar/app-webpack/tsconfig-preset",
+  "compilerOptions": {
+    "baseUrl": "."
+  }
+}
```
Comment (sleberknight) [inline ui/tsconfig.json:6]: make it happy...newline

File: ui/test/cypress/fixtures/example.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (sleberknight) [inline ui/test/cypress/fixtures/example.json:5]: make it happy...newline

File: ui/test/cypress/.gitignore [hunk 8]
```diff
@@ -0,0 +1,2 @@
+videos/*
+screenshots/*
```
Comment (sleberknight) [inline ui/test/cypress/.gitignore:2]: make it happy...newline

File: ui/quasar.extensions.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+      "majestic"
+    ]
+  },
+  "@quasar/testing-e2e-cypress": {
+    "options": [
+      "scripts",
+      "typescript"
+    ]
+  }
+}
```
Comment (sleberknight) [inline ui/quasar.extensions.json:22]: make it happy...newline

File: ui/package.json [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+    "last 8 FirefoxAndroid versions",
+    "last 10 iOS versions",
+    "last 5 Opera versions"
+  ],
+  "engines": {
+    "node": ">= 12.22.1",
+    "npm": ">= 6.13.4",
+    "yarn": ">= 1.21.1"
+  }
+}
```
Comment (sleberknight) [inline ui/package.json:68]: make it happy...newline

**Review by sleberknight — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (chrisrohr) [reply .github/workflows/build.yml:71]: This is OBE, I moved it up to the job level

**Review by sleberknight — APPROVED** [review]

**Pull request merged**